pod.go

```diff
@@ -584,8 +595,8 @@ func (p *Pod) delete() error {
 		return err
 	}
 
-	if state.State != StateReady && state.State != StateStopped {
-		return fmt.Errorf("Pod not ready or stopped, impossible to delete")
+	if state.State != StateReady && state.State != StatePaused && state.State != StateStopped {
```

Comment:
@jodh-intel I understand your point better now: You want to be able to delete a container when it's paused if you either paused it by mistake or if you realize after pausing it that it could be harmful to resume it. Makes sense to me.

@sboeuf So yes, we want to allow for deleting a paused pod.